src/layout/content/story.tsx

```diff
@@ -16,22 +16,24 @@ stories.addDecorator(withKnobs)
 stories.add('default', () => (
```

Comment:
You will have conflict with the MDX branch

Reply:
Indeed, I'll rebase `mdx` this afternoon